### PR TITLE
Added HTML and JS files

### PR DESCRIPTION
**Pull request by albfelix31**
Added HTML and JS files to display a list of users that have posted messages on the webapp.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -25,9 +25,9 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.UUID;
-import java.util.Set;
 import java.util.HashSet;
```
Comment (wayneb0t) [inline src/main/java/com/google/codeu/data/Datastore.java:27]: H < L

**Review by wayneb0t — CHANGES_REQUESTED** [review]
There don't seem to be any HTML or JS files in this pull request. Did you forget to include a commit?

**Comment by albfelix31** [comment]
Sorry! When I pushed it the first time, it seemed to be ignoring the HTML and JS files, because apparently, they were on a gitignore directory. Should be there now. I also fixed the order of the imports on the Datastore file.

**Review by wayneb0t — APPROVED** [review]
Good call pulling the JS into a separate script!

**Pull request merged**